Story 1.json

```diff
@@ -111,7 +111,7 @@
   },
   "100010020130008": {
     "OriginalText": "まもなく惑星マキアの衛星軌道に到着し、\n戦闘部隊員を含む第一次探査隊が\n惑星地表へ降下します。",
-    "Text": "Once we enter Makia's orbit, the primary\nexploration team—including combat personnel—\nmay then safely descend to the surface.",
+    "Text": "Once we enter Makia's orbit, the primary\nexploration team, including combat personnel,\nmay then safely descend to the surface.",
```

Comment:
In the patch, "-" is not a valid character, so it just combines "team" and "including" into "teamincluding".